build.py

```diff
@@ -1317,49 +1315,42 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
         # )
         # trtllm_buildscript = importlib.util.module_from_spec(spec)
         # exec(response.content, trtllm_buildscript.__dict__)
-        # df += trtllm_buildscript.create_postbuild(
-        #     backends[be] # repo tag
-        # )
+        # df += trtllm_buildscript.create_postbuild(backends[be])
+
         df += """
 WORKDIR /workspace
-
 # Remove previous TRT installation
 RUN apt-get remove --purge -y tensorrt* libnvinfer*
 RUN pip uninstall -y tensorrt
-
 # Install new version of TRT using the script from TRT-LLM
 RUN apt-get update && apt-get install -y --no-install-recommends python-is-python3
-RUN git clone --single-branch --depth=1 -b {} https://{}:{}@gitlab-master.nvidia.com/ftp/tekit_backend.git tensorrtllm_backend
+RUN git clone --single-branch --depth=1 -b {} https://github.com/triton-inference-server/tensorrtllm_backend.git tensorrtllm_backend
+RUN cd tensorrtllm_backend && git submodule set-url -- tensorrt_llm https://github.com/NVIDIA/TensorRT-LLM.git
```

Comment:
Unfortunately we won't be able to use the postbuild function right now since we will need to update the script to change the submodule url in the future. Reverted this change back.